opentreemap/treemap/instance.py

```diff
@@ -274,6 +274,9 @@ def set_config(self, value):
     mobile_api_fields = _make_config_property('mobile_api_fields',
                                               DEFAULT_MOBILE_API_FIELDS)
 
+    web_detail_fields = _make_config_property('web_detail_fields',
```

Comment:
While testing I created a new UDF, set its editability permissions, and was briefly puzzled by the fact that I did not see it appear on the tree detail page. We now have to visit 3 management pages to add a field, one to create, one to adjust role permissions, and one to add it to the detail page. There is never a reason to add a UDF and not display it to anyone so I think we need to do something to help the manager understand what they need to do.

Some ideas:

- Automatically add the new field to the fields list.
- Add another notification "bubble."

<img width="253" alt="screen shot 2016-10-17 at 3 13 06 pm" src="https://cloud.githubusercontent.com/assets/17363/19457733/858a4ea8-947c-11e6-98f7-fbff3b42ff7c.png">